### PR TITLE
Adding interface new kpi playbook

### PR DESCRIPTION
**Pull request by bmadhavi74**

### CODE REVIEW

File: community_supplied/Interface/interface-kpis-community.playbook [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * Playbook contains multiple rules which monitor interfaces and notifies when
+ * anomalies are found.
+ * 
+ * 1) Rule check-ethernet-pause-frame-counters-netconf, detects interface ethernet 
+ *    pause frame counters and notifies when anomalies are found.
+ * 2) Rule check-if-mtu-netconf, Detects interface MTU values and notifies when 
+ *    anomalies are found.
+ */
+ healthbot {
+    playbook interface-kpis-community-playbook {
```
Comment (gvarao) [inline community_supplied/Interface/interface-kpis-community.playbook:11]: change playbook name to interface-kpis-additional-playbook

**Review by gvarao — CHANGES_REQUESTED** [review]

**Comment by bmadhavi74** [comment]
Changed the playbook name.

File: community_supplied/Interface/interface-kpis-additional.playbook [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ * Playbook contains multiple rules which monitor interfaces and notifies when
+ * anomalies are found.
+ * 
+ * 1) Rule check-ethernet-pause-frame-counters-netconf, detects interface ethernet 
+ *    pause frame counters and notifies when anomalies are found.
+ * 2) Rule check-if-mtu-netconf, Detects interface MTU values and notifies when 
+ *    anomalies are found.
+ */
+ healthbot {
+    playbook interface-kpis-additional-playbook {
```
Comment (gvarao) [inline community_supplied/Interface/interface-kpis-additional.playbook:11]: change playbook name from "interface-kpis-additional-playbook" to "interface-additional-kpis-playbook"

**Review by gvarao — CHANGES_REQUESTED** [review]

File: community_supplied/Interface/interface-kpis-additional.playbook (hunk 2 above)
Comment (bmadhavi74) [reply community_supplied/Interface/interface-kpis-additional.playbook]: changed the playbook name to "interface-additional-kpis-playbook"

**Review by gvarao — APPROVED** [review]

**Pull request merged**